install.sh

```diff
@@ -158,7 +188,16 @@ then
         Version="tags/${Version}"
     fi
 else
-    Version="tags/${Version}"
+    if [[ -e ".git/refs/tags/${Version}" ]]
```

Comment:
I think this'll always fail because it's not in the Rustlings directory at this time (for normal invocations of the script)